src-tauri/src/platform_specific/linux.rs

```diff
@@ -3,6 +3,81 @@
 use regex::Regex;
 use std::process::Command;
 
+fn get_proton_dir() -> Option<String> {
+    let steam_dir = steamlocate::SteamDir::locate()?;
+    let compat_dir = format!("{}/compatibilitytools.d/", steam_dir.path.display());
+
+    Some(compat_dir)
+}
+
+/// Downloads and installs NS proton
+/// Assumes Steam install
+pub fn install_ns_proton() -> Result<(), thermite::prelude::ThermiteError> {
+    // Get latest NorthstarProton release
+    let latest = thermite::core::latest_release()?;
+
+    let temp_dir = std::env::temp_dir();
+    let path = format!("{}/nsproton-{}.tar.gz", temp_dir.display(), latest);
+    let archive = std::fs::File::create(path.clone())?;
+
+    // Download the latest Proton release
+    log::info!("NorthstarProton download started");
+    thermite::core::download_ns_proton(latest, archive)?;
+    log::info!("NorthstarProton download finished");
+
+    let compat_dir = get_proton_dir().unwrap();
```

Comment:
Maybe write the directory to log. At least in testing on my laptop I had no idea where it was being installed to. Wanted to check whether it picked up the Flatpak install correctly ^^"

Reply:
Good idea, will add.